mackup/config.py

```diff
@@ -25,6 +25,51 @@
     import ConfigParser as configparser
 
 
+def make_config_file(storage_type=ENGINE_DROPBOX, path="",
+                     directory=MACKUP_BACKUP_PATH, whitelist=None,
+                     blacklist=None):
+    """
+    Writes a configuration file (~/.mackup.cfg) using the parameters. If no
+    options are specified, a default configuration file is made.
+
+    Args:
+        (str): The storage_type to be written (dropbox, google_drive,
+        file_system)
+        (str): The path to use if storage_type was file_system
+        (str): The custom directory name that will be used
+        (list): The list of applications that will be synced
+        (list): The list of applications that will not be synced
+    """
+    # Get the names of the apps to check the whitelist and blacklist
+    if not whitelist:
+        whitelist = []
+    if not blacklist:
+        blacklist = []
+    # Make sure the storage_type is correct
+    assert storage_type in (ENGINE_DROPBOX, ENGINE_FS,
+                            ENGINE_GDRIVE)
+    # If the path is specified, make sure it exists
+    if path:
+        assert os.path.exists(path), (
+            "The path {} does not exist!".format(path))
+    # Write the configuration file in the home directory
+    configuration = ["[storage]",
```

Comment:
Can you use Python's ConfigParser to create this file content? Way easier and solid than trying to build it with strings.